contracts/external/cw-token-swap/src/contract.rs

```diff
@@ -59,6 +61,7 @@ struct CounterpartyResponse<'a> {
     pub counterparty: CheckedCounterparty,
     pub other_counterparty: CheckedCounterparty,
     pub storage: Item<'a, CheckedCounterparty>,
+    pub other_storage: Item<'a, CheckedCounterparty>,
```

Comment:
nit: having only the counterparties storage in this struct has some safety guarantees as it means that code with this object can't mutate the other counterparty. not specifying the messages to be executed in `Fund` should mean that we don't need this.